### PR TITLE
[MAINT] update ORCID

### PR DESCRIPTION
**Pull request by Remi-Gau**
- Relates to #3966 

Changes proposed in this pull request:

- update several orcids in citation.cff

### CODE REVIEW

File: CITATION.cff [hunk 1]
```diff
@@ -309,6 +320,7 @@ authors:
     family-names: Wiesner
     website: https://github.com/JohannesWiesner
     affiliation: Central Institute of Mental Health, Mannheim, Germany
+    orcid: https://orcid.org/0000-0001-7040-3516
```
Comment (Remi-Gau) [inline CITATION.cff:323]: @JohannesWiesner

is that your ORCID?

File: CITATION.cff [hunk 2]
```diff
@@ -460,10 +478,12 @@ authors:
     family-names: Gensollen
     website: https://nicolasgensollen.github.io/
     affiliation: Inria, Paris, France
+    orcid: https://orcid.org/0000-0001-7199-9753
```
Comment (Remi-Gau) [inline CITATION.cff:481]: @nicolasgensollen

is that your ORCID?

Comment (NicolasGensollen) [reply CITATION.cff:481]: @Remi-Gau Yes it is ! 👍 Thanks for this ! 🙏

File: CITATION.cff [hunk 3]
```diff
@@ -630,6 +657,7 @@ authors:
     family-names: Mzayek
     website: https://github.com/ymzayek
     affiliation: Inria, Saclay, France
+    orcid: https://orcid.org/0000-0003-2904-2530
```
Comment (Remi-Gau) [inline CITATION.cff:660]: forgot to ask but @ymzayek, this is your ORCID?

Comment (ymzayek) [reply CITATION.cff:660]: Yes!

File: CITATION.cff (hunk 1 above)
Comment (JohannesWiesner) [reply CITATION.cff:323]: Yes it is, thanks!

**Pull request merged**